packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6038,16 +6026,11 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   private requestScreenShareFloor() {
-    if (
-      !this.mediaProperties.hasLocalShareTrack() ||
-      !this.mediaProperties.mediaDirection.sendShare
-    ) {
+    if (!this.mediaProperties.shareVideoStream || !this.mediaProperties.mediaDirection.sendShare) {
```

Comment:
we should be using hasLocalShareStream() here

Reply:
Done in #3053

Reply:
it's done incorrectly there, but I'll mark this comment here as resolved and we can continue the discusion in #3053